### DISCUSSION TITLE
Match any string or number

### DISCUSSION THREAD

**Discussion by Nikola-Milovic** [pashagolub/pgxmock#70] [Q&A] [answered]
How can I use `AnyArg` to match any string or number? As I generate tokens and I don't know their exact values

```
mock.ExpectExec("INSERT INTO refresh_tokens").WithArgs(1, pgxmock.AnyArg(), pgxmock.AnyArg())
```

EDIT: previously was a different question but it didn't make much sense so I am reusing this one as I cannot delete it

**Comment by pashagolub** [comment, marked as answer]
It seems you're doing everything correct. Do you still have any issues with it?

**Reply by Nikola-Milovic** [reply]
I don't remember honestly, I thought the issue was that I didn't use the `AnyArg` correctly

**Reply by pashagolub** [reply]
Ah, ok :) Sorry for the delay with answer. Please, reopen this discussion if you'll need to clarify something.
